birthdays-project/BirthdayApp/admin.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib import admin
+from django.db import models
+from django.forms import TextInput, Textarea
+
+from .models import User, Birthdate
+
+
+# Modifies the appearance of the textfields in the admin section.
+class ModAdmin(admin.ModelAdmin):
```

Comment:
Due to the change from TextFields to CharFields, I was able to remove this code all together.